src/llmq/context.cpp

```diff
@@ -20,7 +20,7 @@
 #include <masternode/sync.h>
 
 LLMQContext::LLMQContext(CChainState& chainstate, CConnman& connman, CEvoDB& evo_db, CSporkManager& sporkman, CTxMemPool& mempool,
-                         const std::unique_ptr<PeerManager>& peerman, bool unit_tests, bool wipe) :
+                         const std::unique_ptr<CJContext>& cj_ctx, const std::unique_ptr<PeerManager>& peerman, bool unit_tests, bool wipe) :
```

Comment:
cj_ctx is used only by ehfSignalsHandler -> cj_ctx can be drop from here (see other comments)

Reply:
Resolved in latest push.